prettyprinter/src/Data/Text/Prettyprint/Doc/Internal.hs

```diff
@@ -1882,6 +1887,65 @@ renderShowS = \sds -> case sds of
     SAnnPush _ x -> renderShowS x
     SAnnPop x    -> renderShowS x
 
+-- * Debugging
+--
+-- Use the @pretty-simple@ package to get a nicer layout for 'show'n
+-- 'Diag's
+--
+-- >>> Text.Pretty.Simple.pPrintNoColor . diag $ align (vcat ["foo", "bar"])
+-- Column 
+--    [ 
+--        ( 10
+--        , Nesting 
+--            [ 
+--                ( 10
+--                , Cat ( Text 3 "foo" ) 
+--                    ( Cat ( FlatAlt Line Empty ) ( Text 3 "bar" ) )
+--                ) 
+--            ]
+--        ) 
+--    ]
```

Comment:
These haddocks don't actually show up in the generated docs, I believe because the exports look like this:

https://github.com/quchen/prettyprinter/blob/7da3b1d32f8a74efb3bcf1b2d062f8f24a64c918/prettyprinter/src/Data/Text/Prettyprint/Doc/Internal.hs#L18-L20

Is there a reason why we can't simply remove the export list and use implicit exports here?

Reply:
https://github.com/quchen/prettyprinter/commit/7574eeb08b867c60536fc214fcaf9032feef126c indicates that this style was used to appease HLint. 

https://github.com/quchen/prettyprinter/pull/107 will show whether HLint still rejects implicit imports.